README.rst

```diff
@@ -6,10 +6,6 @@ gitchangelog
    :target: https://pypi.python.org/pypi/gitchangelog/
    :alt: Latest PyPI version
 
-.. image:: https://img.shields.io/pypi/dm/gitchangelog.svg?style=flat
-   :target: https://pypi.python.org/pypi/gitchangelog/
-   :alt: Number of PyPI downloads
-
```

Comment:
Please remove this change, shields.io downloads from pypi is working fine

Reply:
done, but this MR is already 1,5 years old, no hope it will be merged ;)